diff --git a/airootfs/etc/mkinitcpio.conf.d/archiso.conf b/airootfs/etc/mkinitcpio.conf.d/archiso.conf
index 5c008e5..d690fca 100644
--- a/airootfs/etc/mkinitcpio.conf.d/archiso.conf
+++ b/airootfs/etc/mkinitcpio.conf.d/archiso.conf
@@ -1,3 +1,3 @@
-HOOKS=(base udev microcode modconf kms memdisk archiso archiso_loop_mnt archiso_pxe_common archiso_pxe_nbd archiso_pxe_http archiso_pxe_nfs block filesystems keyboard)
+HOOKS=(base udev microcode modconf kms keyboard keymap memdisk archiso archiso_loop_mnt archiso_pxe_common archiso_pxe_nbd archiso_pxe_http archiso_pxe_nfs block filesystems)
 COMPRESSION="xz"
 COMPRESSION_OPTIONS=(-9e)
